php/php.dbgp/src/org/netbeans/modules/php/dbgp/packets/RunResponse.java

```diff
@@ -57,4 +70,18 @@ private void updateBreakpointsView(DebugSession session, String id) {
         }
     }
 
+    private void setCurrentBreakpointText(DebugSession session, String message, String code) {
+        DebugSession.IDESessionBridge bridge = session.getBridge();
+        if (bridge != null) {
+            BreakpointModel breakpointModel = bridge.getBreakpointModel();
+            if (breakpointModel != null && breakpointModel.isSearchCurrentBreakpointById()) {
+                AbstractBreakpoint bp = breakpointModel.getCurrentBreakpoint(session);
+                if (bp != null && bp instanceof ExceptionBreakpoint) {
```

Comment:
```suggestion
                if (bp instanceof ExceptionBreakpoint) {
```

The check for `null` is not needed.

Reply:
Done.